### PR TITLE
Feat: Add code of conduct for  project

### PR DESCRIPTION
**Pull request by KimVianney**
Update CONTRIBUTING.md by adding CODE of CONDUCT

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+This isn’t an exhaustive list of things that you can’t do. Rather, take it in the spirit in which it’s intended - a guide to make it easier to enrich all of us and the technical communities in which we participate.
+
+This code of conduct applies to all spaces managed by the NoseBleedSection project. This includes any forums created by the project team which the community uses for communication. In addition, violations of this code outside these spaces may affect a person's ability to participate within them.
+
+If you believe someone is violating the code of conduct, we ask that you report it by emailing <>
```
Comment (Kudzmat) [inline CODE_OF_CONDUCT.md]: We don't have an email to report to yet so maybe remove this for now.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    - Personal insults, especially those using racist or sexist terms.
+        Unwelcome sexual attention.
+
+    - Advocating for, or encouraging, any of the above behavior.
+
+    - Repeated harassment of others. In general, if someone asks you to stop, then stop.
+
+- **When we disagree, try to understand why**. Disagreements, both social and technical, happen all the time and NoseBleedSection is no exception. It is important that we resolve disagreements and differing views constructively. Remember that we’re different. The strength of NoseBleedSection comes from its varied community, people from a wide range of backgrounds. Different people have different perspectives on issues. Being unable to understand why someone holds a viewpoint doesn’t mean that they’re wrong. Don’t forget that it is human to err and blaming each other doesn’t get us anywhere. Instead, focus on helping to resolve issues and learning from mistakes.
+
+Original text courtesy of the Speak Up! project.
```
Comment (Kudzmat) [inline CODE_OF_CONDUCT.md:32]: Do we need to keep this to avoid plagiarism, or can we remove it?

**Review by Kudzmat — COMMENTED** [review]
Thanks, Vianney, this looks great. I just made a few comments you can get back to me about, but I'm happy with this overall!

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (KimVianney) [reply CODE_OF_CONDUCT.md]: Alright no problem

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (KimVianney) [reply CODE_OF_CONDUCT.md:32]: I think leaving it there is kind of a reference that we used someone else's info, therefore I dont think that counts as plagiarism. So I think we should leave it there

**Review by Kudzmat — APPROVED** [review]

**Pull request merged**